pkg/validations/cluster_node.go

```diff
@@ -0,0 +1,89 @@
+package validations
+
+import (
+	"fmt"
+)
+
+const (
+	singleAZCount = 1
+	MultiAZCount  = 3
+)
+
+// MinReplicasValidator is responsible for verifying whether the minReplicas value adheres to specific rules:
+//
+// * The minReplicas value must be a positive number.
+// * If multiple availability zones are used, the Cluster must have a minimum of 3 compute nodes and the total count of nodes must be a multiple of 3.
+// * If a single availability zone is used, the Cluster must include at least 2 compute nodes.
+// * For Hosted clusters, the number of compute nodes must be a multiple of the number of private subnets.
+func MinReplicasValidator(minReplicas int, multiAZ bool, isHostedCP bool, privateSubnetsCount int) error {
+	if minReplicas <= 0 {
```

Comment:
For reviewer:
Please notice this was changed from `if minReplicas < 0`
Because the error message indicates it can't be 0
See source: https://github.com/openshift/rosa/blob/4f38019e5767ec0d460bde065c2228c7e457c9cd/cmd/create/cluster/cmd.go#L2592

Reply:
The error message is wrong, replicas can be 0, to support autoscaling, they just can not be a negative int.

Reply:
Ok, changed it back and changed the error message accordingly